app/src/main/java/com/example/mycalculator/MainActivity.java

```diff
@@ -2,13 +2,100 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.annotation.SuppressLint;
 import android.os.Bundle;
+import android.view.View;
+
+import com.example.mycalculator.databinding.ActivityMainBinding;
+import com.example.mycalculator.mvp.contract.CalculatorContract;
+import com.example.mycalculator.mvp.model.CalculatorModel;
+import com.example.mycalculator.mvp.presenter.CalculatorPresenter;
+import com.example.mycalculator.mvp.view.CalculatorView;
+import com.example.mycalculator.utils.StringUtils;
 
 public class MainActivity extends AppCompatActivity {
 
+    private ActivityMainBinding binding;
+    private CalculatorContract.CalculatorPresenterContract presenter;
+
+    @SuppressLint("WrongViewCast")
```

Comment:
why did you add this?

Reply:
@SuppressLint("WrongViewCast") this?

Reply:
yes

Reply:
If i remeber correctly, when i was doing the calculator layout, i think i got an error of the lint tool, and to solve that problem i used a quick fix with alt*enter. Regardless of that, following your comment, i erased it and now it´s no more, and the app still works.

Reply:
okay, try to remove it ;)